HopsanGUI/Widgets/AnimationWidget.cpp

```diff
@@ -433,6 +432,10 @@ void AnimationWidget::play()
 //! @todo The plot data object in container should perhaps be cleared of these generations
 void AnimationWidget::playRT()
 {
+    if(mpSpeedSpinBox->value() < 0) {
```

Comment:
I have not had time to test or look at the code in detail, but what happens if you change the speed during real-time simulation, to a negative value.
Will it stop, or just create lots of errors ?

Maybe limiting the lowest value to zero in RT simulation where the value goes into to drive the simulation.

Reply:
It will simply not simulate at all in RT with negative speed. If setting it to negative during RT animation, the animation will stop at the current position and the time will start rewinding without anything happening. If switching back to positive, the animation will continue once the animation time catches up with the last working animation time. I think we can live with this for now, as changing the limitations depending on animation type would be complicated.